api/routes/socket.js

```diff
@@ -23,7 +23,12 @@ module.exports = function (app, server) {
 				socketController.test(action.message);
 				break;
 			case 'socket/ticket/new':
-				socketController.addTicket(action.ticket);
+				try {
+					socketController.addTicket(action.ticket);
+				} catch (error) {
+					console.error(error);
```

Comment:
Nuh-uh.

Reply:
What should we do in these cases?

Reply:
Well, first this whole function has to be made `async` and `addTicket` call `await`ed for. Secondly, the `this` here won’t even refer to anything useful because of arrow functions, so this code path is (I would assume) untested.

Honestly I’d just ignore the error for now and put a TODO here.